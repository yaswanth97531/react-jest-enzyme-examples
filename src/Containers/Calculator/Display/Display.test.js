import Display from './Display';

let wrapper;

beforeEach(() => {
wrapper = shallow(<Display />);
});

describe('<Display /> rendering', () => {
  const tree = renderer.create( <Display displayValue={''} / > ).toJSON();
  it('should render correctly', () => expect(tree).toMatchSnapshot());
  
  it('should render the <div> Component', () => {
    expect(wrapper.find('div').length).toEqual(1);
  });

  it('renders the value of displayValue', () => {
    wrapper.setProps({ displayValue: 'test' });
    expect(wrapper.text()).toEqual('test');
  });
});
describe('<Display /> interactions', () => {});
describe('<Display /> lifecycle method invocations', () => {});