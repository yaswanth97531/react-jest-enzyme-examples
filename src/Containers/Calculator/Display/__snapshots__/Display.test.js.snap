// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> rendering should render correctly 1`] = `
<div
  className="display-container"
>
  <p
    className="display-value"
  >
    
  </p>
</div>
`;
